lib/features/email/domain/usecases/preview_email_from_eml_file_interactor.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:core/presentation/state/failure.dart';
+import 'package:core/presentation/state/success.dart';
+import 'package:core/utils/platform_info.dart';
+import 'package:dartz/dartz.dart';
+import 'package:tmail_ui_user/features/email/domain/model/preview_email_eml_request.dart';
+import 'package:tmail_ui_user/features/email/domain/repository/email_repository.dart';
+import 'package:tmail_ui_user/features/email/domain/state/preview_email_from_eml_file_state.dart';
+import 'package:tmail_ui_user/features/email/presentation/model/eml_previewer.dart';
+
+class PreviewEmailFromEmlFileInteractor {
```

Comment:
what is this mean of `File`?

Reply:
Handles viewing of the contents of an EML file.